rpc/src/rpc.rs

```diff
@@ -1871,10 +1997,22 @@ impl JsonRpcRequestProcessor {
             Error::invalid_params("Invalid param: mint could not be unpacked".to_string())
         })?;
 
-        let interest_bearing_config = mint
-            .get_extension::<InterestBearingConfig>()
-            .map(|x| (*x, bank.clock().unix_timestamp))
-            .ok();
+        let interest_bearing_config = OptionFuture::from(
+            mint.get_extension::<InterestBearingConfig>()
+                .map(|interest_bearing_config| {
+                    let bank = Arc::clone(&bank);
+                    async move {
+                        let unix_timestamp = self
+                            .runtime
+                            .spawn_blocking(move || bank.clock().unix_timestamp)
```

Comment:
I don't think that this should go in the bg pool? The clock sysvar account is
smol and pretty much guaranteed to be cached